### PR TITLE
dead people don't appear upright on cameras anymore

### PR DESCRIPTION
**Pull request by tralezab**
image

## About The Pull Request

🎉 

## Why It's Good For The Game

🎉

### CODE REVIEW

**Comment by Twaticus** [comment]
i love you i love you

**Comment by RaveRadbury** [comment]
Does this work for alive people that are laying down?

**Comment by RaveRadbury** [comment]
Does this work for living people that are laying down?

**Comment by AnturK** [comment]
Fix the cast issue and it's good to go.

**Comment by ShizCalev** [comment]
cast issue?

File: code/modules/photography/camera/camera_image_capturing.dm [hunk 1]
```diff
@@ -69,11 +73,13 @@
 		var/xo = (A.x - center.x) * world.icon_size + A.pixel_x + xcomp
 		var/yo = (A.y - center.y) * world.icon_size + A.pixel_y + ycomp
 		if(ismovableatom(A))
-			var/atom/movable/AM = A
-			xo += AM.step_x
-			yo += AM.step_y
+			var/obj/effect/appearance_clone/clone = A
```
Comment (nemvar) [inline code/modules/photography/camera/camera_image_capturing.dm]: This is in a loop, in case you were wondering why the cast doesn't work.

Comment (tralezab) [reply code/modules/photography/camera/camera_image_capturing.dm]: yep, they're not pretty

**Pull request merged**